client/src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import AppBar from "@material-ui/core/AppBar";
+import PermIdentityIcon from "@material-ui/icons/PermIdentityOutlined";
+import Toolbar from "@material-ui/core/Toolbar";
+import "./styles.css";
```

Comment:
Let's add our css filed to the `styles` folder, with the name of the file being the corresponding component/page.